### PR TITLE
Update civicrm_state_province.tpl

### PR DESCRIPTION
**Pull request by gah242s**
-- Add missing Pakistani Province (Punjab)

### CODE REVIEW

**Comment by kurund** [comment]
@gah242s Can you please file an issue for this in http://issues.civicrm.org. The changes done by you handles new install case. We might also want to handle the same for upgrades. You need to add the similar sql to 4.7.alpha1.mysql.tpl. If you need any help feel free to ping us on #civicrm (irc channel)

**Comment by gah242s** [comment]
Added PR for upgrade
https://github.com/civicrm/civicrm-core/pull/6665

**Comment by yashodha** [comment]
closing in favor of #6669

**Comment by gah242s** [comment]
Now wait.  I go out of my way to create all of the stuff, carefully documenting it to the best of my newbie abilities, and the checker says everything is good.  Then 2 days later, @yashodha comes in, creates the IDENTICAL patch on a new ticket that happens to fail checks, and closes the original ticket.  What did I do wrong?

File: xml/templates/civicrm_state_province.tpl [hunk 1]
```diff
@@ -3994,3 +3994,6 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1081, "SJ", "Samtskhe-Javakheti"),
 (NULL, 1081, "SK", "Shida Kartli"),
 (NULL, 1074, "C", "Central");
```
Comment (yashodha) [inline xml/templates/civicrm_state_province.tpl]: There should be a comma instead of a semi-colon here and closing semi-colon at line no 3999.

**Comment by gah242s** [comment]
Ok.  I see that now and get what happened.  Why were the changes not added to this ticket and a new one created?  Did I need to do something different there?

**Pull request closed without merging**